config/shapes/channel_selection.py

```diff
@@ -325,3 +356,89 @@ def channel_selection(channel, era, special=None):
         return Selection(name="ee", cuts=cuts)
     else:
         raise ValueError("Given special selection does not exist")
+    
+
+def tau_id_channel_selection(channel, era, vs_jet_wp="tight", vs_ele_wp="vvloose"):
```

Comment:
whats the difference here compared to the `channel_selection` function ?

Reply:
I removed this code duplication